templates/email/oxfordshire/alert-update.txt

```diff
@@ -0,0 +1,32 @@
+Subject: New [% site_name %] updates on report: '[% report.title %]'
```

Comment:
This txt template does not include a link to the video anywhere. But see my other comment about which template to use - if that comment is approved, you could put the link in the equivalent _council_reference_alert_update.txt

Reply:
@dracos see comment above. But I wasn't sure what to do with this file, I noticed when I didn't include it I was getting an error when loading the email on my browser.

Reply:
It's for the text part of the email - for clients that don't handle HTML email, you need to include all the contents from the HTML part here also. So just a link to the YouTube video somewhere would presumably be enough.

Reply:
Thanks for clarifying, Matthew.  I just included some info + the link.